test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -258,8 +281,10 @@ void ProtoConverter::visit(VarDecl const& _x)
 	m_output << "let " << varName << " := ";
 	visit(_x.expr());
 	m_output << "\n";
-	m_scopeVars.top().push_back(varName);
-	m_variables.push_back(varName);
+	if (m_inFunctionDef)
+		m_funcVars.back().back().push_back(varName);
+	else
+		m_globalVars.back().push_back(varName);
```

Comment:
Same for `m_globalVars` being non-empty here?

Reply:
Nevermind, just re-read the `open*Scope` functions

Reply:
Hmm... thanks! :) I somehow thought I was missing a check for vector emptiness somewhere.

It looks like I added asserts on vector validity over at #7765